console-server/image.yaml

```diff
@@ -1,18 +1,18 @@
 schema_version: 1
 
 version: 1.4
-name: amq7/amq-online-1-console-httpd
-description: "AMQ Online image for the AMQ Online Console HTTPD"
+name: amq7/amq-online-1-console-server-rhel7
```

Comment:
Will we be appending -rhel7 (-rhel8) to all future image names?

Reply:
Yes, this is automatically appended when creating image requests in comet, so we have no choice right now.